storage/src/main/java/tech/pegasys/artemis/storage/ChainStorageClient.java

```diff
@@ -48,13 +56,40 @@
 
   public ChainStorageClient(EventBus eventBus) {
     this.eventBus = eventBus;
+    transactionCommitter = new TransactionCommitter(eventBus);
```

Comment:
(nit) we might as well make this this.transactionCommitter?

Reply:
Going to follow up with another PR that changes this anyway to make the APIs a bit nicer. Minimising conflicts by not changing this here. :)